Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -692,3 +680,27 @@ FVector USpatialActorChannel::GetActorSpatialPosition(AActor* Actor)
 		return FVector::ZeroVector;
 	}
 }
+
+void USpatialActorChannel::SpatialViewTick()
+{
+	if (Actor != nullptr && !Actor->IsPendingKill() && IsReadyForReplication())
+	{
+		bool bOldNetOwned = bNetOwned;
+		bNetOwned = Actor->GetNetConnection() != nullptr;
+		if (bFirstTick || bOldNetOwned != bNetOwned)
+		{
+			bool bSuccess = true;
+
+			if (NetDriver->IsServer())
+			{
+				bSuccess &= Sender->UpdateEntityACLs(Actor, GetEntityId());
+			}
+			else
+			{
+				Sender->SendComponentInterests(Actor, GetEntityId());
+			}
+
+			bFirstTick = !bSuccess;
```

Comment:
Hmmm, this is probably not clearly readable. Maybe a comment above or a more explicit if-else?